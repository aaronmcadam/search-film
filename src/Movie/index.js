import React from 'react';
import styled from 'styled-components';

const Wrapper = styled.div`
  background-color: #fff;
  border-radius: 5px;
  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.14), 0 1px 5px 0 rgba(0, 0, 0, 0.12),
    0 3px 1px -2px rgba(0, 0, 0, 0.2);
  color: #000;
  margin: 0.5rem 0.5rem 0 0;
  width: 300px;
`;

const MovieImage = styled.img`border-radius: 5px 5px 0 0;`;
const MovieTitle = styled.h2`
  font-size: 1.5rem;
  margin: 0;
  margin-bottom: 0.5rem;
`;
const MovieContent = styled.div`
  border-radius: 0 0 2px 2px;
  padding: 1.5rem;
`;
const MovieOverview = styled.div``;

export default function Movie({ poster_path, title, overview }) {
  return (
    <Wrapper>
      <MovieImage src="http://via.placeholder.com/300x450/4A1479/D0806A?text=Poster" />
      <MovieContent>
        <MovieTitle>Blade Runner</MovieTitle>
        <MovieOverview>Cyberpunk classic</MovieOverview>
      </MovieContent>
    </Wrapper>
  );
}
