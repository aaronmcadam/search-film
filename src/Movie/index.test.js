import React from 'react';
import { shallow } from 'enzyme';
import Movie from '.';

const setup = () => {
  const wrapper = shallow(<Movie />);

  return {
    wrapper,
  };
};

test('renders', () => {
  const { wrapper } = setup();

  expect(wrapper).toMatchSnapshot();
});
