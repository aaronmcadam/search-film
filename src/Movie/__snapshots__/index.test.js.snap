// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<styled.div>
  <styled.img
    src="https://image.tmdb.org/t/p/w300_and_h450_bestv2undefined"
  />
  <styled.div>
    <styled.h2 />
    <styled.div />
  </styled.div>
</styled.div>
`;
